### ISSUE TITLE
Make public keys available at /.well-known/jwks.json

### ISSUE THREAD

**Issue by byewokko** [TeskaLabs/seacat-auth#104]
OAuth recommends that the authorization server's public keys be available at `/.well-known/jwks.json`. This can be done by a new proxy location in nginx config.

**Comment by elpablos** [comment]
setup for NGINX:
```
    # jwk
    location /.well-known/jwks.json {
        proxy_pass http://seacat-pub/openidconnect/public_keys;
    }
 ```

**Comment by byewokko** [comment]
resolved in #249
